templates/statsd_mapping.yaml.erb

```diff
@@ -0,0 +1,3 @@
+<% require 'yaml' -%>
+<% full_config = { 'mappings'=>@mappings } -%>
+<%= full_config.to_yaml -%>
```

Comment:
Can't we use this puppet function to generate the yaml? https://github.com/puppetlabs/puppetlabs-stdlib#to_yaml
and provide that to the file as `content`? That would allow us to not use a template at all.

Reply:
I looked at code for other exporter how they did it and they also used this method, thought it was a bit silly too but didn't know about the to_yaml function in the stdlib :-)

The [blackbox](https://github.com/voxpupuli/puppet-prometheus/blob/master/templates/blackbox_exporter.yaml.erb) and [process](https://github.com/voxpupuli/puppet-prometheus/blob/master/templates/process-exporter.yaml.erb) exporter do something similiar.